### ISSUE TITLE
Region Zoning

### ISSUE THREAD

**Issue by andrewmcwatters** [Planimeter/vertex-adventure#9]
![](http://i.imgur.com/Y7MMmFu.png)

**Comment by ryan-kingstone** [comment]
We could perhaps have the starting location near the water, ie a port town?

**Comment by andrewmcwatters** [comment]
Yeah, that would be really cool. I like the feel of doing something like that, but I worry about how we'll have to contain the player for a short period of time to put them through a tutorial.

I don't want to use one of the four islands for that, so it might be better to put new players on a short quest that forces them to learn things. The duality here is that it's one quest they can complete, too, thus contributing to their quest completion count/percentage.

**Comment by ryan-kingstone** [comment]
We could send them along a "rail" of discovery-investigation type quest in order to make them learn all the mechanics and so on.
